core/src/main/java/de/danoeh/antennapod/core/feed/Feed.java

```diff
@@ -25,6 +29,9 @@
     public static final String TYPE_RSS2 = "rss";
     public static final String TYPE_ATOM1 = "atom";
     public static final String PREFIX_LOCAL_FOLDER = "antennapod_local:";
+    public static final String SUPPORT_INTERNAL_SPLIT = "\n";
+    public static final String SUPPORT_INTERNAL_EQUAL = "\t";
```

Comment:
How about using less common characters here, like the ASCII record separator (`\u001e`) and group separator (`\u001f`)? Those can then be safely removed from the input string, so feed creators cannot cause issues in AntennaPod.